src/Network/NWBrowser.cs

```diff
@@ -45,22 +45,20 @@ public NWBrowser (NWBrowserDescriptor descriptor, NWParameters parameters)
 			if (descriptor == null)
 				throw new ArgumentNullException (nameof (descriptor));
 
-			InitializeHandle (nw_browser_create (descriptor.Handle, parameters?.Handle ?? IntPtr.Zero));
+			InitializeHandle (nw_browser_create (descriptor.Handle, parameters?.GetHandle () ?? IntPtr.Zero));
```

Comment:
-> simply `parameters.GetHandle ()`
that's an extension method that does the same, i.e. `return self == null ? IntPtrZero : self.Handle;`